jenkins/job-configs/kubernetes-jenkins-pull/credentials.yaml

```diff
@@ -0,0 +1,7 @@
+- wrapper:
+    name: e2e-credentials-binding  # TODO(fejta): make this match non-pull
```

Comment:
./kubernetes-jenkins-pull/credentials.yaml and ./kuberentes-jenkins/credentials.yaml should both combine into a single ./credentials.yaml